android/app/src/main/java/com/hotwirerndemo/RNSession.kt

```diff
@@ -1,27 +1,36 @@
 package com.hotwirerndemo
 import android.content.Context
 import android.util.Log
+import android.webkit.JavascriptInterface
 import android.widget.FrameLayout
 import androidx.appcompat.app.AppCompatActivity
+import com.facebook.react.bridge.Arguments
 import com.facebook.react.bridge.ReactContext
+import com.facebook.react.bridge.WritableMap
+import com.facebook.react.uimanager.events.RCTEventEmitter
 import dev.hotwire.turbo.session.TurboSession
-import dev.hotwire.turbo.views.TurboView
 import dev.hotwire.turbo.views.TurboWebView
-import dev.hotwire.turbo.visit.TurboVisit
 import java.util.*
+import org.json.JSONObject
 
 class RNSession(context: Context) : FrameLayout(context) {
 
+    private val reactContext = context as ReactContext
+    private val registeredVisitableViews = mutableListOf<SessionSubscriber>()
+
     val session: TurboSession = run {
```

Comment:
Rename to turboSession https://github.com/software-mansion-labs/react-native-turbo-demo/pull/15#discussion_r993119067